src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -1061,4 +1061,13 @@ public void testModuleNameInMultiThreadMode() throws Exception {
         }
     }
 
+    @Test
+    public void testJacocoWorkaround() throws Exception {
+        // This test verifies nothing. It is a workaround for the Jacoco limitations.
+        // see https://github.com/jacoco/jacoco/issues/117 for more details.
```

Comment:
Please move this to javadoc like we define in https://github.com/checkstyle/checkstyle/issues/4970 and add a breif description of what the issue is in jacoco.